packages/manager/apps/telecom/src/components/telecom/telephony/scheduler/telephony-scheduler.less

```diff
@@ -48,6 +48,22 @@
   }
 }
 
+.telecom-scheduler-bank-holidays {
+  .year-list .oui-radio-toggle-group {
+    flex-direction: column;
+
+    .oui-radio-toggle,
+ .oui-radio-toggle:first-of-type,
+ .oui-radio-toggle:last-of-type {
```

Comment:
Why is this needed?

Reply:
The UI originally had the `uib-btn-radio` displayed as a column (meaning the buttons, which denotes the years like 2018, 2019 etc are displayed one below the other).  But the `oui-radio-toggle` component does not support this vertical orientation. Only horizontal orientation of the buttons is supported, due to which, the borders are not visible for the right side of the first button, both sides of the buttons in the middle and the left side of the last button. Here we changed the orientation of the buttons to vertical and hence we will have to have uniform borders for each of the buttons, which is what has been done :)